mlflow/utils/databricks_utils.py

```diff
@@ -151,6 +151,16 @@ def get_cluster_id():
     return spark_session.conf.get("spark.databricks.clusterUsageTags.clusterId")
 
 
+def get_job_group_id():
+    try:
+        dbutils = _get_dbutils()
+        job_group_id = dbutils.entry_point.getJobGroupId()
```

Comment:
I found this function [here](https://github.com/databricks/universe/blob/9676a1c16f1c3fdb10da3f19fe01df907602ef42/spark/driver/python/python_shell/dbruntime/dbutils.py#L141) and decided to use it. Let me know if more options are needed.